### PR TITLE
[W5.4][T15-A4]Calvin Tantio

### PR DESCRIPTION
**Pull request by CT15**
Ready for review

### CODE REVIEW

**Comment by archthegit** [comment]
Everything looks good. But i noticed in one place you imported java.util.* i don't know if that'll make your program a little slower

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.commands;
+
+public class SortCommand extends Command{
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/SortCommand.java]: Need header comments for the class and its methods (if the methods are doing some non-trivial stuff)

File: src/seedu/addressbook/data/AddressBook.java [hunk 2]
```diff
@@ -128,4 +128,11 @@ public boolean equals(Object other) {
                         && this.allPersons.equals(((AddressBook) other).allPersons)
                         && this.allTags.equals(((AddressBook) other).allTags));
     }
+
+    /**
+     * Sort the address book according to names in alphabetical order
```
Comment (okkhoy) [inline src/seedu/addressbook/data/AddressBook.java]: *Sorts* the address book ...

**Comment by okkhoy** [comment]
@CT15 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**